backend/api/serializers/preset.py

```
@@ -7,6 +7,32 @@
+            raise serializers.ValidationError(serializer.errors)
+            
+        # Validiere Inhalt (Whitelist etc.) via Service
+        from api.services.dynamic_statistik_service import DynamicStatistikService
+        try:
+            is_valid, error_message = DynamicStatistikService.validate_query(
+                base_model=value.get('base_model'),
+                filters=value.get('filters', {}),
+                group_by=value.get('group_by')
+            )
+            if not is_valid:
+                raise serializers.ValidationError(error_message)
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)